react/features/base/icons/svg/index.js

```diff
@@ -42,6 +43,8 @@ export { default as IconInfo } from './info.svg';
 export { default as IconInvite } from './invite.svg';
 export { default as IconKick } from './kick.svg';
 export { default as IconLiveStreaming } from './public.svg';
+export { default as IconMeetingLocked } from './meeting-locked.svg';
```

Comment:
Can't see the icons now, but are these any different than lock room and unlock room?

Reply:
yep, they are different
![image](https://user-images.githubusercontent.com/9974975/81295012-08682000-9070-11ea-83fa-ada69a18d4a9.png)